### PR TITLE
Pass `damlc build` logs through daml logger

### PR DESCRIPTION
**Pull request by cocreature**
This gives us some timings which can be useful in some
circumstances (I want timings around data-dependencies next).

I also changed the default log level to INFO. The `damlc build` output
doesn’t get any noisier from that.

Example output:

```
2021-11-01 15:25:17.16 [INFO]  [build]  []
Compiling foobar to a DAR.

2021-11-01 15:25:17.80 [INFO]  [build]  []
Created .daml/dist/foobar-0.0.1.dar
```

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: compiler/damlc/lib/DA/Cli/Damlc.hs [hunk 1]
```diff
@@ -453,7 +453,7 @@ execIde telemetry (Debug debug) enableScenarioService options =
                   then Logger.Debug
                   -- info is used pretty extensively for debug messages in our code base so
                   -- I've set the no debug threshold at warning
```
Comment (sofiafaro-da) [inline compiler/damlc/lib/DA/Cli/Damlc.hs:455]: This comment should be updated or removed.

**Review by sofiafaro-da — APPROVED** [review]
That sounds very useful, thanks!

File: compiler/damlc/lib/DA/Cli/Damlc.hs (hunk 1 above)
Comment (cocreature) [reply compiler/damlc/lib/DA/Cli/Damlc.hs]: good catch, just dropped it. I think INFO is the usual default so doesn’t really need an explanation.

**Pull request merged**